### PR TITLE
Bugfix: Fixes zos_copy and zos_fetch deprecation msg for using _play_context.verbosity

### PR DESCRIPTION
**Pull request by ddimatos**
Same as PR #806 but for release version 1.6.0.

##### SUMMARY
This addresses issue #779 where a deprecation warning would appear when zos_copy or zos_fetch ran. Of the two modules, only the `zos_copy` module  could be automated with a test because its plugin is invoked immediately where as with `zos_fetch` its invoked last and that can not work for testing display.verbosity.

In addition to this, the test case for #616 was addressed here because the original test case could not run properly on local work stations and the pipeline. So a new test case was written to run on both.

The pytest-ansible fixture was updated to support the `zos_copy` adhoc test case.

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by rexemin** [comment]
Screenshot 2023-06-19 at 20 01 30
Screenshot 2023-06-19 at 20 01 45

**Comment by ddimatos** [comment]
Thank you @rexemin your help getting this through was much appreciated.

**Review by AndreMarcel99 — APPROVED** [review]
Ready

File: changelogs/fragments/814-zos_data_set-update-vsam-copy.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+bugfixes:changelogs/fragments/814-zos_data_set-update-vsam-copy.yml
```
Comment (fernandofloresg) [inline changelogs/fragments/814-zos_data_set-update-vsam-copy.yml:1]: This will break change log generation

**Review by fernandofloresg — CHANGES_REQUESTED** [review]

File: changelogs/fragments/814-zos_data_set-update-vsam-copy.yml (hunk 1 above)
Comment (ddimatos) [reply changelogs/fragments/814-zos_data_set-update-vsam-copy.yml]: All fixed

**Review by fernandofloresg — APPROVED** [review]
🦭 of approval.

**Pull request merged**